.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "bracketSpacing": true,
+    "endOfLine": "lf",
+    "singleQuote": true,
+    "tabWidth": 2,
+    "useTabs": false
+}
```

Comment:
Perfect, thanks!

Reply:
Very well! I noticed when I made the PR that it was outdated but I closed the previous one. I'm going to keep an eye for sure and thank you so much! It's really encouraging